Gemfile

```diff
@@ -51,6 +53,7 @@ group :development, :test do
   gem 'listen' # Listens to file modifications
   gem 'letter_opener' # Preview mail in the browser instead of sending
   gem 'ffaker' # A library for generating fake data such as names, addresses, and phone numbers
+  gem 'faker' # A library for generating more fake data
```

Comment:
Should we remove FFaker then? I think it is better to only keep one, thus reducing the number of external dependencies (less maintenance) and avoiding the confusion of "which faker should I use for my use case")

Reply:
I remove `Faker` because Phong might be using `FFaker`. Added constant OAuthFake.Google instead to solve issue with redundancy.